components/sidebar.tsx

```diff
@@ -23,15 +23,11 @@ import { useSettings } from "@/contexts/settings-context";
 
 const navigation = [
   { name: "Network Status", href: "/", icon: Network },
-  { name: "Account Portal", href: "/account-portal", icon: LayoutDashboard },
+  { name: "Account Portal", href: "/account-portal", icon: LayoutDashboard, needAuth: true },
```

Comment:
Don't put this back, we agreed 2 days ago, we will show this and that we will show login button instead of Add Funds on the account portal.